src/hooks/loadables/useLoadSafeMessages.ts

```diff
@@ -1,104 +1,21 @@
 import { useEffect } from 'react'
+import { getSafeMessages } from '@gnosis.pm/safe-react-gateway-sdk'
+import type { SafeMessageListPage } from '@gnosis.pm/safe-react-gateway-sdk'
 
 import useAsync from '@/hooks/useAsync'
 import { logError, Errors } from '@/services/exceptions'
 import useSafeInfo from '@/hooks/useSafeInfo'
-import { SafeMessageListItemType, SafeMessageStatus } from '@/store/safeMessagesSlice'
 import type { AsyncResult } from '@/hooks/useAsync'
-import type { SafeMessageListPage } from '@/store/safeMessagesSlice'
-
-const MOCK_DATA: SafeMessageListPage['results'] = [
-  {
-    type: SafeMessageListItemType.DATE_LABEL,
-    timestamp: Date.now(),
-  },
-  {
-    type: SafeMessageListItemType.MESSAGE,
-    messageHash: '0x123',
-    status: SafeMessageStatus.NEEDS_CONFIRMATION,
-    logoUri: '',
-    name: 'Example dApp',
-    message: 'Example string message',
-    creationTimestamp: Date.now(),
-    modifiedTimestamp: Date.now(),
-    confirmationsSubmitted: 1,
-    confirmationsRequired: 2,
-    confirmations: [
-      {
-        owner: { value: '0x123', name: 'John' },
-        signature: '0x123',
-      },
-    ],
-    proposedBy: {
-      value: '0x123',
-      name: 'John',
-    },
-    preparedSignature: null,
-  },
-  {
-    type: SafeMessageListItemType.MESSAGE,
-    messageHash: '0x456',
-    status: SafeMessageStatus.NEEDS_CONFIRMATION,
-    logoUri: '',
-    name: 'Example dApp',
-    message: {
-      example: 'Typed data message',
-    },
-    creationTimestamp: Date.now(),
-    modifiedTimestamp: Date.now(),
-    confirmationsSubmitted: 1,
-    confirmationsRequired: 2,
-    confirmations: [
-      {
-        owner: { value: '0x123', name: 'Alice' },
-        signature: '0x123',
-      },
-    ],
-    proposedBy: {
-      value: '0x123',
-      name: 'Alice',
-    },
-    preparedSignature: '0x123',
-  },
-  {
-    type: SafeMessageListItemType.MESSAGE,
-    messageHash: '0x456',
-    status: SafeMessageStatus.CONFIRMED,
-    logoUri: '',
-    name: 'Example dApp',
-    message: {
-      example: 'Typed data message',
-    },
-    creationTimestamp: Date.now(),
-    modifiedTimestamp: Date.now(),
-    confirmationsSubmitted: 1,
-    confirmationsRequired: 1,
-    confirmations: [
-      {
-        owner: { value: '0x123', name: 'Alice' },
-        signature: '0x123',
-      },
-    ],
-    proposedBy: {
-      value: '0x123',
-      name: 'Alice',
-    },
-    preparedSignature: '0x123',
-  },
-]
 
 export const useLoadMessages = (): AsyncResult<SafeMessageListPage> => {
   const { safe, safeAddress, safeLoaded } = useSafeInfo()
 
+  // Re-fetch when chainId/address
```

Comment:
nit: This comment just describes the dependency array and is not necessary imo

Reply:
This mirrors our other loading hooks.

Reply:
These comments mirror our other polling hooks. However, I agree with you and have adjusted it accordingly.